### PR TITLE
`PipelineRun` timeouts is in Beta

### PR DESCRIPTION
**Pull request by jerop**
# Changes

`PipelineRun` timeouts were promoted to Beta. This change removes
the listing of `PipelineRun` timeouts from the alpha features.

Issue: https://github.com/tektoncd/pipeline/issues/4460
PR: https://github.com/tektoncd/pipeline/pull/4813

/kind misc

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [n/a] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ n/a [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [x] Release notes block below has been filled in (if there are no user facing changes, use release note "NONE")

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -413,20 +413,19 @@ the `feature-flags` ConfigMap alongside your Tekton Pipelines deployment.
...
+| [Step and Sidecar Overrides](./taskruns.md#overriding-task-steps-and-sidecars)                        | [TEP-0094](https://github.com/tektoncd/community/blob/main/teps/0094-specifying-resource-requirements-at-runtime.md)       |                                                                      |                             |
+| [Matrix](./matrix.md)                                                                                 | [TEP-0090](https://github.com/tektoncd/community/blob/main/teps/0090-matrix.md)                                            |                                                                      |                             |
+| [Embedded Statuse](pipelineruns.md#configuring-usage-of-taskrun-and-run-embedded-statuses)            | [TEP-0100](https://github.com/tektoncd/community/blob/main/teps/0100-embedded-taskruns-and-runs-status-in-pipelineruns.md) |                                                                      |                             |
```
Comment (lbernick) [inline docs/install.md:428]: there's a typo in "Statuses" that you can fix while you're here

**Review by lbernick — APPROVED** [review]

File: docs/install.md (hunk 1 above)
Comment (jerop) [reply docs/install.md]: saw that, will fix separately

**Review by afrittoli — COMMENTED** [review]
/approve

**Pull request merged**